Index: Andrei Hnedchik.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>data = 'Не знаю, как там в Лондоне, я не была. Может, там собака друг человека. А у нас управдом друг человека!'\r\nprint('Шаг 1 - Количество символов -', len(data))\r\nprint('Шаг 2 - Развернутая строка -',data[::-1])\r\nprint('Шаг 3 - Слова с большой буквы -',data.title())\r\nprint('Шаг 4 - Текст с прописными буквами -',data.upper())\r\n\r\none, two, tree = data.count('нд'), data.count('ам'), data.count('о')\r\nprint(f'Шаг 5 - Количесво вхождений \"нд\" = {one}, количесво вхождений \"ам\" = {two}, количесво вхождений \"о\" = {tree}')\r\nprint('Шаг 6 - Текст с удаленными пробелами -',data.replace(' ', ''))\r\nprint('Шаг 7 - Поиск по индексу слова \"Может\" - ',data.index(\"Может\"))\r\n\r\nrule = data.split()\r\nprint('Шаг 8 - Развернутое предложение -', *rule[::-1])\r\nprint('Шаг 9 - Исходная строка -', data)\r\n# hello
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Andrei Hnedchik.py b/Andrei Hnedchik.py
--- a/Andrei Hnedchik.py	(revision 4099936040eb84fd44cb905ec7e52e5c5f763a77)
+++ b/Andrei Hnedchik.py	(date 1717168937700)
@@ -11,5 +11,4 @@
 
 rule = data.split()
 print('Шаг 8 - Развернутое предложение -', *rule[::-1])
-print('Шаг 9 - Исходная строка -', data)
-# hello
\ No newline at end of file
+print('Шаг 9 - Исходная строка -', data)
\ No newline at end of file
